### ISSUE TITLE
Error installing vampnet 

### ISSUE THREAD

**Issue by javanasse** [hugofloresgarcia/vampnet#1]
I am installing `vampnet` on a 2021 Macbook Pro running OS Monterey 12.3

I am able to clone and install `audiotools` and `descript`.
When I run `pip install -e ./vampnet` as described in the README, I get a fatal `Repository not found` error for `'https://github.com/hugofloresgarcia/audiotools.git/'`.

Full error message follows.

```
>>> pip install -e ./vampnet 
Obtaining file:///path/to/parent/vampnet
  Preparing metadata (setup.py) ... done
Collecting audiotools@ git+https://github.com/hugofloresgarcia/audiotools.git (from vampnet==0.0.1)
  Cloning https://github.com/hugofloresgarcia/audiotools.git to /private/var/folders/jr/h2z3gbpd0jz4ktdfjxjs6wf00000gn/T/pip-install-_gg4j_9c/audiotools_fd9572e2a98a4f5fa90d9e9538617d16
  Running command git clone --filter=blob:none --quiet https://github.com/hugofloresgarcia/audiotools.git /private/var/folders/jr/h2z3gbpd0jz4ktdfjxjs6wf00000gn/T/pip-install-_gg4j_9c/audiotools_fd9572e2a98a4f5fa90d9e9538617d16
  remote: Repository not found.
  fatal: repository 'https://github.com/hugofloresgarcia/audiotools.git/' not found
  error: subprocess-exited-with-error
  
  × git clone --filter=blob:none --quiet https://github.com/hugofloresgarcia/audiotools.git /private/var/folders/jr/h2z3gbpd0jz4ktdfjxjs6wf00000gn/T/pip-install-_gg4j_9c/audiotools_fd9572e2a98a4f5fa90d9e9538617d16 did not run successfully.
  │ exit code: 128
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× git clone --filter=blob:none --quiet https://github.com/hugofloresgarcia/audiotools.git /private/var/folders/jr/h2z3gbpd0jz4ktdfjxjs6wf00000gn/T/pip-install-_gg4j_9c/audiotools_fd9572e2a98a4f5fa90d9e9538617d16 did not run successfully.
│ exit code: 128
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

**Comment by javanasse** [comment]
It seems that [this line](https://github.com/hugofloresgarcia/vampnet/blob/a84c25c272432ce9aa05e04c9bf1b2cda8d916c7/setup.py#L33) and its immediate successor are references to repos that don't exist or are private.

```
        "audiotools @ git+https://github.com/hugofloresgarcia/audiotools.git",
        "lac @ git+https://github.com/hugofloresgarcia/lac.git"
```

**Comment by javanasse** [comment]
Is the `lac` referenced here the same `lac` from [here](https://github.com/baidu/lac/tree/master/python), as in (L)exical (A)nalysis of (C)hinese?

**Comment by hugofloresgarcia** [comment]
Ah! Sorry, seem to have missed to update those requirements. Should be in fixed now in 3445a716606e07f38aa0687d331c587f4eb7a9e0

**Comment by hugofloresgarcia** [comment]
Closing for now, but feel free to reopen if the issue persists!
